### DISCUSSION TITLE
Mixed-content on some pages in cp

### DISCUSSION THREAD

**Discussion by chrsgrffth** [statamic/cms#4764] [Troubleshooting] [answered]
Hi, I recently rewrote an old website that was running Statamic v1 to Statamic v3 and deployed it to a DO App. Most things look good, but I am running into the issue with Mixed Content on a few pages in the cp (specfically `/collections/pages`).

My `APP_URL` and `ASSET_URL` are both set with `https://` and I've added code to try to force https in production (the solution I've seen some previous discussions had for this issue):

```
if (App::environment('production')) {
  URL::forceScheme('https');
}
```

I'm wondering what I could be missing or have configured incorrectly? Thanks!

**Comment by jasonvarga** [comment, marked as answer]
Are you using Cloudflare? Or some other kind of proxy?

If Cloudflare, you can install https://github.com/monicahq/laravel-cloudflare
Otherwise, you can configure your proxy: https://laravel.com/docs/8.x/requests#configuring-trusted-proxies

**Reply by chrsgrffth** [reply]
https://laravel.com/docs/8.x/requests#configuring-trusted-proxies

This worked for me. Thank you! 

I ended up setting:
```
protected $proxies = '*';
```
in `app/Http/Middleware/TrustProxies.php`
